src/components/calltree/ProfileCallTreeContextMenu.js

```diff
@@ -310,6 +320,13 @@ class ProfileCallTreeContextMenu extends PureComponent<Props> {
             ? 'Focus on calls made by this function'
             : 'Focus on function'}
         </MenuItem>
+        <MenuItem
+          onClick={this.handleClick}
+          data={{ type: 'collapse-function-subtree' }}
+        >
+          <span className="profileCallTreeContextMenuIcon profileCallTreeContextMenuIconCollapse" />
+          {"Collapse function's subtree across the entire tree"}
```

Comment:
> I have no way to type the nice quote directly

Wikipedia tells me, on a Mac US Layout: `Option + Shift + ]`
Or just `&apos;` as a HTML entity.

> and have never used it in my English writing

From Wikipedia: `Both simplifications carried over to computer keyboards and the ASCII character set. However, although these are widely used due to their ubiquity and convenience, they are deprecated in contexts where proper typography is important.`

> not sure why we should make it more complicated than it needs to be.

complicated? Why complicated?
It's nicer, and maybe more important, it's proper typography. The `'` is just a convenience from a time when we were using typewriters or using 7-bit ASCII.
And here I took the time to go to Wikipedia to be able to give you the character to use anyway (`’`), so you just can copy paste it :)

Reply:
Embedding this in a review so that you don't miss it ^